extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -46,6 +47,7 @@ function init(
 
 	overrideSlotService(slotRegistry, legacyBtfBlocker);
 	updatePageLevelTargeting(legacyContext, pageLevelTargeting, skin);
+	syncSlotsStatus(slotRegistry, context.get('slots'));
```

Comment:
Should we sync it on each pv on mobile wiki?

Reply:
https://github.com/Wikia/app/pull/15191/files#diff-da84cd8d83f4dd1e7a6ae474ba85fe86R59